ocs_ci/templates/ocs-deployment/multicluster/openshift_dr_hub_operator.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
   name: odr-hub-operator
   namespace: openshift-dr-system
 spec:
-  channel: stable-4.9
+  channel: stable-4.10
```

Comment:
Do we need to support multiple versions?
E.g. 4.10 for OCP/ODF 4.10
4.11 for OCP/ODF 4.11 ?
If so I think we should move this to OCP or ODF version config file I think as this will be automatically setting proper version based on --ocp-version or --ocs-version and will use proper version of odr-hub

Reply:
As was discussed on the DR meeting, it should depend on OCS/ODF version so let's move this to OCS version config file please.

Reply:
done